Python/Tests/IronPython/IronPythonAnalysisTest.cs

```diff
@@ -1,517 +1,517 @@
-            var entry = ProcessText(@"
-import wpf
-from System.Windows import Window
-w = Window()
-w.Activate
-");
-
-            var result = entry.GetValuesByIndex("w.Activate", 1).ToArray();
-            Assert.AreEqual(1, result.Length);
-            Console.WriteLine("Docstring was: <{0}>", result[0].Documentation);
-            Assert.IsFalse(string.IsNullOrWhiteSpace(result[0].Documentation));
-        }
-
-        /*
-        [TestMethod, Priority(0)]
-        public void OverrideParams() {
-            var text = @"
-import System
-
-class MyArrayList(System.Collections.ArrayList):
-    def AddRange(self, col):
-        x = col
-";
-            var entry = ProcessText(text);
-            var x = entry.GetMembersFromName("x", text.IndexOf("x = col")).ToSet();
-            AssertUtil.ContainsExactly(x, GetMembers(ClrModule.GetPythonType(typeof(System.Collections.ICollection)), true));
-        }*/
-
-
-        /// <summary>
-        /// Verify importing wpf will add a reference to the WPF assemblies
-        /// </summary>
-        [TestMethod, Priority(0)]
-        public void WpfReferences() {
-            var entry = ProcessText(@"
-import wpf
-from System.Windows.Media import Colors
-");
-
-            AssertUtil.Contains(entry.GetMemberNamesByIndex("Colors", 1), "Blue");
-            AssertUtil.Contains(entry.GetMemberNamesByIndex("wpf", 1), "LoadComponent");
-        }
-
-        [TestMethod, Priority(0)]
-        public void XamlEmptyXName() {
-            // [Python Tools] Adding attribute through XAML in IronPython application crashes VS.
-            // http://pytools.codeplex.com/workitem/743
-            using (var analyzer = PythonAnalyzer.CreateSynchronously(InterpreterFactory, Interpreter)) {
-                string xamlPath = TestData.GetPath(@"TestData\Xaml\EmptyXName.xaml");
-                string pyPath = TestData.GetPath(@"TestData\Xaml\EmptyXName.py");
-                var xamlEntry = analyzer.AddXamlFile(xamlPath);
-                var pyEntry = analyzer.AddModule("EmptyXName", pyPath);
-
-                xamlEntry.ParseContent(new FileStreamReader(xamlPath), null);
-
-                using (var parser = Parser.CreateParser(new FileStreamReader(pyPath), PythonLanguageVersion.V27, new ParserOptions() { BindReferences = true })) {
-                    pyEntry.UpdateTree(parser.ParseFile(), null);
-                }
-
-                pyEntry.Analyze(CancellationToken.None);
-            }
-        }
-
-        private static string[] GetMembers(object obj, bool showClr) {
-            var dir = showClr ? ClrModule.DirClr(obj) : ClrModule.Dir(obj);
-            int len = dir.__len__();
-            string[] result = new string[len];
-            for (int i = 0; i < len; i++) {
-                Assert.IsTrue(dir[i] is string);
-                result[i] = dir[i] as string;
-            }
-            return result;
-        }
-
-    }
-}
+﻿/* ****************************************************************************
```

Comment:
I'm not sure why this shows up as changed. When I have both branches checked out locally and compare the files directly, they're byte for byte identical (i.e. there's no line ending differences or tab differences). Suggestions?